### ISSUE TITLE
Nothing happens when `spqr-coordinator  run --unknown-parameter`

### ISSUE THREAD

**Issue by Denchick** [pg-sharding/spqr#419]
**How to reproduce**

Literally,

```
root@denchick /home/denchick # /usr/bin/spqr-coordinator run --unknown-parameter
root@denchick /home/denchick #
```

**Expected behavior**
Print an error `parameter X is unknown` and exit

**Comment by Denchick** [comment]
I found something really interesting and impressing

```
[QA]root@klg-2r2gqwj46eh8ysjx /home/denchick # spqr-coordinator status
2024/01/22 09:29:49 Running config: {
  "log_level": "fatal",
  "qdb_addr": "http://localhost:2379",
  "http_addr": "klg-2r2gqwj46eh8ysjx.db.yandex.net:7003",
  "addr": "klg-2r2gqwj46eh8ysjx.db.yandex.net:7002",
  "shard_data": ""
}
{"level":"debug","address":"http://localhost:2379","client":824637370304,"time":"2024-01-22T09:29:49+03:00","message":"etcdqdb: NewEtcdQDB"}
{"level":"info","time":"2024-01-22T09:29:49+03:00","message":"running coordinator app"}
{"level":"debug","address":"klg-2r2gqwj46eh8ysjx.db.yandex.net:7003","time":"2024-01-22T09:29:49+03:00","message":"etcdqdb: try coordinator lock"}
{"level":"debug","raft-term":2,"lease-id":7587876051021473467,"time":"2024-01-22T09:29:49+03:00","message":"etcd keep alive channel"}
{"level":"debug","address":"klg-2r2gqwj46eh8ysjx.db.yandex.net:7003","time":"2024-01-22T09:29:50+03:00","message":"etcdqdb: try coordinator lock"}
{"level":"debug","raft-term":2,"lease-id":7587876051021473467,"time":"2024-01-22T09:29:50+03:00","message":"etcd keep alive channel"}
{"level":"error","error":"qdb is already in use","time":"2024-01-22T09:29:50+03:00","message":"qdb already taken, waiting for connection"}
{"level":"debug","raft-term":2,"lease-id":7587876051021473473,"time":"2024-01-22T09:29:51+03:00","message":"etcd keep alive channel"}
{"level":"debug","address":"klg-2r2gqwj46eh8ysjx.db.yandex.net:7003","time":"2024-01-22T09:29:51+03:00","message":"etcdqdb: try coordinator lock"}
{"level":"debug","raft-term":2,"lease-id":7587876051021473467,"time":"2024-01-22T09:29:51+03:00","message":"etcd keep alive channel"}
{"level":"error","error":"qdb is already in use","time":"2024-01-22T09:29:51+03:00","message":"qdb already taken, waiting for connection"}
{"level":"debug","raft-term":2,"lease-id":7587876051021473485,"time":"2024-01-22T09:29:52+03:00","message":"etcd keep alive channel"}
...
```

**Comment by Denchick** [comment]
Solved in #436
